apps/nova/src/pages/Register.tsx

```diff
@@ -86,15 +86,16 @@ const Register: React.FC = () => {
         return mutation.mutate({
           ...payload,
           role,
-          callbackUrl: "http://localhost:5173/verify-email", // todo: should be dynamic
+          callbackUrl:
+            BackendPath[import.meta.env.VITE_BACKEND] + Route.VerifyEmail,
```

Comment:
This is a wrong `callbackUrl`. Take a look at this comment here https://github.com/litespace-org/litespace/pull/17#discussion_r1808618060 to understand how the `callbackUrl` works.